sdk/core/Azure.Core.Experimental/src/Azure.Core.Experimental.csproj

```diff
@@ -20,4 +20,8 @@
     <Compile Include="$(AzureCoreSharedSources)NullableAttributes.cs" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Curious: what is this used for in this PR?

Reply:
Removed